### PR TITLE
.github: enhance the user story template

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

**Comment by elzinko** [comment]
What about adding : 
- wireframes if front involved
- api contracts

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 1]
```diff
@@ -21,8 +18,27 @@ assignees: ''
...
+- [ ] Acceptance criterias are defined
+- [ ] The individual responsible for validating the user story is assigned
+- [ ] The user story can be shown off
+- [ ] Does not depend on an missing feature
+- [ ] Negociable with implementors
+- [ ] Valuable to the end users
+- [ ] Estimable by the implementors
+- [ ] Small enough to fit in a sprint
+- [ ] Testable by implementors
+
```
Comment (elzinko) [inline .github/ISSUE_TEMPLATE/user_story.md]: what about adding : 
- api contracts
- wireframes if front involved

**Comment by elzinko** [comment]
also : 
- The story should be written exactly in the ‘user story’ format.

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 2]
```diff
@@ -21,8 +18,27 @@ assignees: ''
 
 <!-- What is the point of this user story ? Why is it necessary ? -->
 
+## Definition of ready
+
+<!-- All these items are checked by an implementation team so it can begin work -->
+<!-- DO NOT CHECK THESE YOURSELF -->
+
+- [ ] The user story is well defined
+- [ ] Acceptance criterias are defined
```
Comment (elzinko) [inline .github/ISSUE_TEMPLATE/user_story.md]: what about : "Acceptance criteria must be understood by the team"

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 3]
```diff
@@ -21,8 +18,27 @@ assignees: ''
...
+- [ ] The individual responsible for validating the user story is assigned
+- [ ] The user story can be shown off
```
Comment (elzinko) [inline .github/ISSUE_TEMPLATE/user_story.md]: what about : 
"The team should understand how to provide a demo of the features"

File: .github/ISSUE_TEMPLATE/user_story.md (hunk 1 above)
Comment (multun) [reply .github/ISSUE_TEMPLATE/user_story.md]: I don't think the API contracts have anything to do with the definition of ready: it's not something the one that writes the user story can do anything about, it's up to us

File: .github/ISSUE_TEMPLATE/user_story.md (hunk 3 above)
Comment (multun) [reply .github/ISSUE_TEMPLATE/user_story.md]: "The team should understand how to demonstrate the features"

**Review by eckter — APPROVED** [review]

**Review by flomonster — APPROVED** [review]

**Pull request merged**